core/job/service/job_service.go

```diff
@@ -72,12 +75,18 @@ type JobService struct {
 	logger log.Logger
 }
 
+type AlertManager interface {
+	io.Closer
```

Comment:
We are not using `Close()` function in this file. so we should not have this as part of contract.